src/views/Home.vue

```diff
@@ -214,12 +214,10 @@ export default defineComponent({
     });
 
     Mousetrap.bind(
-      ["enter"],
+      "shift+enter",
```

Comment:
I changed the `Add Audio Cell` hotkey to `shift+enter` since enter is not caught by the default behavior. Also when focusing the audio cell the same action is also triggered by `shift+enter`, this could help reducing the user's confusion.

Reply:
Thanks for the suggestion. This one is better.